src/containers/SubmitSteps/Questionary/Step1/Step1.tsx

```diff
@@ -70,11 +115,65 @@ const Step1 = ({
       <TitleBlack>{t('questionary:title')}</TitleBlack>
       <QuestionNote>{t('questionary:note')}</QuestionNote>
       <WomanWithPhone />
+      <ProgressIndicator
+        currentStep={metadata?.current}
+        totalSteps={metadata?.total}
+        progressBar
+      />
+      <QuestionText extraSpace first>{t('questionary:whenPcrTest')}
+      </QuestionText>
+      <Controller
+        control={control}
+        name="pcrTestDate"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <DatePicker
+            label="Date"
+            value={value ? new Date(value) : null}
+            locale={i18n.language}
+            onChange={onChange}
+          />
+        )}
+      />
+
+      <QuestionText extraSpace>
+        {t('questionary:resultPcrTest.question')}
+      </QuestionText>
+      <Controller
+        control={control}
+        name="pcrTestResult"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <OptionList
+            singleSelection
+            value={{ selected: value ? [value] : [] }}
+            onChange={v => onChange(v.selected[0])}
+            items={[{
+              value: 'positive',
+              label: t('questionary:resultPcrTest.options.positive'),
+            },
+            {
+              value: 'negative',
+              label: t('questionary:resultPcrTest.options.negative'),
+            },
+            {
+              value: 'pending',
+              label: t('questionary:resultPcrTest.options.pending'),
+            },
+            {
+              value: 'unsure',
+              label: t('questionary:resultPcrTest.options.unsure'),
+            }]}
+          />
+        )}
+      />
+      <p><ErrorMessage errors={errors} name="name" /></p>
```

Comment:
Better to use the prop "as" 
```suggestion
      <ErrorMessage errors={errors} name="name" as="p" />
```
or use the render method to render the `<p></p>`
to keep the dom clear when there is no error for "name" (Instead of having a empty p tag)